app/uk/gov/hmrc/emailgateway/connector/DownstreamConnector.scala

```diff
@@ -53,25 +53,30 @@ class DownstreamConnector @Inject()(httpClient: HttpClientV2) extends Logging {
                 HttpEntity.Streamed(response.bodyAsSource, None, response.header(CONTENT_TYPE))
               )
             }
-            .recover {
-              case _: Throwable =>
-                BadGateway(toErrorResponse("REQUEST_DOWNSTREAM", "An issue occurred when the downstream service tried to handle the request"))
+            .recoverWith { case t: Throwable =>
+              Future.successful(
+                BadGateway("{\"code\": \"REQUEST_DOWNSTREAM\", \"desc\": \"An issue occurred when the downstream service tried to handle the request\"}")
+                  .as(MimeTypes.JSON)
```

Comment:
Think this was probably better as it was before, and it wouldn't need the `.as(MimeTypes.JSON)` because play would automatically know that from the fact it was getting a `JsValue`

```suggestion
                BadGateway(toErrorResponse("REQUEST_DOWNSTREAM", "An issue occurred when the downstream service tried to handle the request"))
```

Reply:
Refactored to use Json implicit writes